guides/source/configuring.md

```diff
@@ -1385,6 +1385,11 @@ Configures deprecation warnings that the Application considers disallowed. This
 
 Allows you to disable all deprecation warnings (including disallowed deprecations); it makes `ActiveSupport::Deprecation.warn` a no-op. This is enabled by default in production.
 
+#### `active_support.local_store_isolation_level`
```

Comment:
Let's document the base level config and the rails 7.0 config in this section: https://edgeguides.rubyonrails.org/configuring.html#results-of-config-load-defaults.

Reply:
Done.